src/api.rs

```diff
@@ -0,0 +1,2 @@
+pub use trussed_core::api::*;
+pub use trussed_core::types::NotBefore;
```

Comment:
nit: I don't think we need to add the pub use here. We can just change the import in fido-authenticator.

Reply:
To make the migration as easy as possible, I tried to keep the changes compatible.  I would suggest to merge it like this and once everything is migrated to trussed-core, we can do a cleanup round on trussed itself.